test_pull_requests

```diff
@@ -2089,7 +2089,8 @@ popd
       # First, get the next jenkins build number
       next_build_num = get_next_build(branch, settings)
 
-      pull_id_params = "&BRANCH=#{branch}"
+      branch_param = settings["branch_param"] || "BRANCH"
```

Comment:
Not sure i follow -- each trigger maps to a specific Jenkins job. I don't see necessarily why we would expect different triggers to share this information as we make no assumptions about the target jobs.